app/assets/stylesheets/forever_style_guide/modules/_nav.scss

```diff
@@ -1,96 +1,107 @@
-/* nav
----------------------------------------------------------------------------- */
-
-$color-nav-bg:  color('white');
-$color-nav-active: color('gray-100');
-$color-nav-hover: color('secondary');
 $header_height: 50px;
-$header_active_border_height: 2px;
+$offcanvas-width: 270px;
+$offcanvas-padding: 15px;
+$nav-bg-color: color('gray-700');
 
-.nav {
-  &.nav-pills {
-    @extend %card-style;
+@mixin nav-border($sides...) {
+  @each $side in $sides {
+    border-#{$side}: 1px solid color('gray-400');
   }
+}
 
-  .nav-item {
-    @include clearfix;
-    font-family: $font-face-regular;
-    font-size: $font-size-300;
-
-    .nav-item-tab,
-    .nav-item-list {
-      border: 0;
-      height: $header_height;
-      color: color('gray-500');
-    }
-    .nav-item-tab {
-      @include border_highlight("bottom", transparent);
+// Basic styles - A lot of overriding Bootstrap defaults
+.navbar-toggle {
+  @media (max-width: $grid-float-breakpoint) {
+    display: block;
+  }
+}
+.navbar-brand {
+  position: absolute;
+  width: 100%;
+  left: 0;
+  top: 0;
+  margin: auto;
+  padding: $padding-small-vertical 0 0;
```

Comment:
In this instance we do need to declare all 4 sides since navbar-brand includes padding: 15px 15px; out of the box from Bootstrap.